Registry.py

```diff
@@ -17,6 +17,8 @@
 
 from flask import request, jsonify, abort, Blueprint
 from threading import Event
+from nmoscommon.auth.nmos_auth import RequiresAuth
```

Comment:
Could this and 'AUTH_MODE' be removed for now until we decide how to do testing of Nodes/Registries etc?